### PR TITLE
Update documentation for Calico operator

### PR DESCRIPTION
**Pull request by caseydavenport**
*Issue #, if available:*

*Description of changes:*

This is a documentation PR that corresponds with the following changes:

- Switch Calico to operator: https://github.com/aws/amazon-vpc-cni-k8s/pull/1297 
- Fix Calico installation issue: https://github.com/aws/amazon-vpc-cni-k8s/pull/1410

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/calico.md
Comment (tmjd) [inline doc_source/calico.md:29]: I think this was suppose to be calico-crs.yaml.

**Review by tmjd — CHANGES_REQUESTED** [review]
One change needed but otherwise LGTM.

**Comment by jimdial-aws** [comment]
I get 404 errors for all the new links in this PR.

**Comment by caseydavenport** [comment]
> I get 404 errors for all the new links in this PR.

@jimdial-aws Yeah, I'm not sure how to order these, so would appreciate guidance on what the right process is. This PR depends on the changes in this one: https://github.com/aws/amazon-vpc-cni-k8s/pull/1410

I think the links probably need to be updated for the next release (v1.8?). The changes in that PR (1410) are against master.

**Comment by jimdial-aws** [comment]
@caseydavenport  Once the other PR changes are merged and these links are live, I'll look at merging this again.  Thanks.

**Comment by jayanthvn** [comment]
I have merged this aws/amazon-vpc-cni-k8s#1410 to master.

**Comment by caseydavenport** [comment]
Thanks @jayanthvn! 

Is there a way to make this docs update in such a way that it will go live with the next release from master (v1.8)?

**Comment by jimdial-aws** [comment]
I'll watch for the change to go live and try to make the docs change as soon as possible, but it may not be until Monday, depending on when the change goes live.

**Comment by jimdial-aws** [comment]
Thank you for your submission!

**Pull request merged**
